src/library/scala/collection/mutable/Shrinkable.scala

```diff
@@ -52,7 +52,6 @@ trait Shrinkable[-A] {
     *  @param xs   the iterator producing the elements to remove.
     *  @return the $coll itself
     */
-  @nowarn("msg=will most likely never compare equal")
```

Comment:
if the compiler's no longer warning on this, I guess we can get rid of it? I'm not sure what changed to stop it warning though, and if that's a bug.

specifically, the things that the compiler doesn't (didn't?) think would compare equal were `this` (a `Shrinkable[_]`) and `xs` (an `IterableOnce[_]`), IIRC

Reply:
ah, hmm. okay, perhaps someone else will choose to dig into it in a separate PR

Reply:
I mean, to be clear, the compiler *is* wrong—many `Shrinkable`s in the library are `IterableOnce` as well. however, the compiler has no way of knowing that (as far as I know?), so I think these are sensible warnings. I'm not sure why the compiler stopped emitting them.